css/styles.css

```diff
@@ -0,0 +1,343 @@
+/**Global Colors & Fonts**/
+:root{
+ --color-white: #ffffff;
+ --color-navy: #093c60;
+ --color-grey: #ececec;
+ --color-green: #81b741;
+ --color-black: #000000;
+
+ --title-font:'Vollkorn', serif;
+ --body-font:'Montserrat', sans-serif;
+}
+
+
+
+/**layout**/
+.row{
+  margin-bottom: 2rem;
+}
+main{
+  max-width: 68.75rem;
+  margin: auto;
+  padding-left:0.75rem;
+  padding-right:0.75rem;
+}
+.d-flex{
+  display: flex;
+}
+
+/**fonts**/
+body, p {
+  font-family: var(--body-font);
+  line-height: 1.75rem;
+}
+h1{
+  font-size:2.25rem;
+  font-family: var(--title-font);
+  color: var(--color-navy);
+  font-weight: bold;
+  margin-bottom:2rem;
+}
+h2{
+  font-size:2rem;
+  font-family: var(--title-font);
+  color: var(--color-navy);
+  font-weight: bold;
+}
+h3{
+  font-size:1.75rem;
+  font-family: var(--title-font);
+  color: var(--color-navy);
+  font-weight: bold;
+}
+h4{
+  font-size:1.5rem;
+  font-family: var(--title-font);
+  color: var(--color-navy);
+}
+h5{
+  font-size:1rem;
+  font-family: var(--title-font);
+}
+h6{
+  font-size:1rem;
+  font-family: var(--title-font);
+  }
+nav a, header nav a{
+  color: var(--color-navy);
+  font-weight: bold;
+}
+
+
+/**Header Section**/
+header{
+  color: var(--color-white);
+}
+header{
+background-image: url(../images/header-image.jpg);
+background-size: cover;
+}
+.header-top{
+  padding-top:4rem;
+  padding-bottom:13.125rem;
+  margin-bottom: 0 !important;
+}
+.header-top, header nav {
+  margin:auto;
+  max-width: 81.25rem;
+}
+header nav ul, footer nav ul{
+  text-align: left;
+  padding-top: 1rem;
+  list-style-type: none;
+  overflow: hidden;
+  margin-bottom: 0rem;
+}
+
+nav ul li{
+  float:left;
```

Comment:
You shouldn't need floats to lay out these elements. `display:inline-block` is a better option